### ISSUE TITLE
Feature request: set a specific mime-type for a hosted file

### ISSUE THREAD

**Issue by dev-tomato** [Raikia/FiercePhish#55]
Although the current implementation does attempt to determine the mime-type of an uploaded file, it would be useful to manually specify an uploaded file's mime-type at upload time to override PHP's automatic detection or to force one for a type that it does not recognize.

**Comment by Raikia** [comment]
Yeah, this will be added shortly as well as better auto-detection of mime types.  I realized this was a problem during testing of the dev branch.
